canonical/greenhouse-browser-scripts
# Add tests and Github actions (WD-2828)

## Done
- Added userscript tests.
- Added Github actions.

## QA
- Run `yarn test` and check that all tests pass.
- Check that Github actions pass.
